Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -30,6 +55,8 @@ public final class ViewAnnotationManager {
     private var expectedHiddenByView: [UIView: Bool] = [:]
     private var viewsByFeatureIds: [String: UIView] = [:]
 
+    private var observers = [ViewAnnotationUpdateObserver]()
```

Comment:
Try storing the observers like this to simplify and speed up the add/remove methods:

```suggestion
    private var observers = [ObjectIdentifier: ViewAnnotationUpdateObserver]()
```

Reply:
Nice tip, thanks!